allauth/account/signals.py

```diff
@@ -9,8 +9,9 @@
 password_changed = Signal(providing_args=["request", "user"])
 password_reset = Signal(providing_args=["request", "user"])
 
-email_confirmed = Signal(providing_args=["email_address"])
-email_confirmation_sent = Signal(providing_args=["confirmation"])
+email_confirmed = Signal(providing_args=["request", "email_address"])
+email_confirmation_sent = Signal(providing_args=["request", "signup",
+                                                 "confirmation"])
```

Comment:
Sorry for nitpicking on details, but I would prefer to have them in order request-confirmation-signup, as this signal is about the confirmation, the signup is only adding extra detail.